src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathFilter.java

```diff
@@ -116,6 +116,9 @@
  * RegexpSinglelineJava
  * </li>
  * <li>
+ * SuppressWarningsHolder
```

Comment:
please remove it from here.
this module do not report any violations now, so it should not be there.
Module will never report violations, Xpath will never be able to suppress violations from it. 

UT might be complaining on absent module, so we need to update UT to skip skip module from consideration as it has no violations, so no need to have support of Xpath for it.

Reply:
@romani Done